### DISCUSSION TITLE
Canal Panda (Portugal)

### DISCUSSION THREAD

**Discussion by ELProductions** [iptv-org/community#790] [Q&A]
### Channel Name

Canal Panda (Portugal)

### Channel ID

CanalPanda.pt

### Website

canalpanda.pt

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
According to our database this channel was closed on 15 December 2022.

https://iptv-org.github.io/channels/pt/CanalPanda
